src/main/java/org/broadinstitute/hellbender/CommandLineProgramTest.java

```diff
@@ -13,6 +16,13 @@
  */
 public abstract class CommandLineProgramTest extends BaseTest {
 
+    public static final String PROPERTY_DATAFLOW_RUNNER = "dataflowRunner";
+
+    static {
+        System.setProperty("spark.driver.allowMultipleContexts", "true");
+        System.setProperty("spark.ui.enabled", "false");
```

Comment:
Should this static block move into BaseTest instead of here?  Unit tests on spark probably rely on it too, so it should probably get moved into the most fundamental test class.  (with an explanatory comment...)